.gitignore

```diff
@@ -4,3 +4,6 @@
 **/target/
 # Tests
 /Kitodo/modules
+# Javadoc
+/javadoc
+/site
```

Comment:
Why are you adding `/site` to be ignored? No file or directory is created.

Reply:
Please, then add a comment that this entry is needed for read the docs if you building them on your local system. Maybe something similar should be added to the JavaDoc entry. As in normal cases both directories does not exists.

Reply:
done: https://github.com/kitodo/kitodo-production/pull/1422/commits/639db94e1d49a75f4b0b4824cd67252784fbb23a